Add section to whitelist headers (#35)
mulesoft/docs-service-mesh: modules/ROOT/pages/download-and-install-service-mesh.adoc

+
image::verified-service-mesh-installer-output.png[]

> === Configure Headers for Client Authorization-based Policies

Reviewer: When used in titles or headings, capitalize the second word of a hyphenated compound exactly as you would if the word stood alone:
```suggestion
=== Configure Headers for Client Authorization-Based Policies
```

Reviewer: If this topic is about whitelisting, shouldn't the title use the term?

Reviewer 2: Good point! Done! Thanks!

> === Whitelist Headers for Client Authorization-Based Policies

Author: ```suggestion
=== Allow Headers for Client Authorization-Based Policies
```

> Anypoint Service Mesh is configured with minimized whitelisted headers for client-based policies:

Reviewer: Are these headers that have been "whitelisted", or "whitelist" (type) headers?

Reviewer 2: there are no whitelist type headers. Whitelisting means "approving" or adding to an approved list.

> Anypoint Service Mesh is configured with minimized whitelisted headers for client-based policies:

Author: ```suggestion
Anypoint Service Mesh is configured with minimized allowed headers for client-based policies:
```

Reviewer 2: Done! Thanks!

> Anypoint Service Mesh is configured with minimal approved (whitelisted) headers for client-based policies:

Author: ```suggestion
Anypoint Service Mesh is configured with minimal approved (allowed) headers for client-based policies:
```

- jwt
- authorization
- client_id
- client-id
- client_secret
- client-secret

> Before use, you must whitelist a custom header. Because header whitelisting is performed at the cluster level, you do not need to whitelist headers for every managed API.

Reviewer: ```suggestion
Before use a custom header, you must whitelist it. Because header whitelisting is performed at the cluster level, you do not need to whitelist headers for every managed API.
```

Reviewer 2: Done! Thanks!

> To whitelist a header, run the following command:

Author: ```suggestion
To allow the use of a header, run the following command:
```

Author: Still not resolved.

> `./service-mesh management config header-whitelist --name=<header name>`

Author: ```suggestion
`asmctl management config policy-headers allow --name=<header name>`
```

Reviewer 2: Done! Thanks!

> `asmctl management config policy-headers allow --name=x-client-secret`

Author: ```suggestion
`asmctl management config policy-headers allow --name=<header-name>`
```

For example, if you want to allow the use of header `x-client-secret`, run:

> `./service-mesh management config header-whitelist add --name=x-client-secret`

Author: ```suggestion
`asmctl management config policy-headers allow --name=x-client-secret`
```

Reviewer 2: Done! Thanks!

> To list the whitelisted headers, run the following command:

Reviewer: ```suggestion
To list existing whitelisted headers, run the following command:
```

Reviewer 2: Done! Thanks!

> To list the whitelisted headers, run the following command:

Author: ```suggestion
To list the allowed headers, run the following command:
```

> `./service-mesh management config header-whitelist list`

Author: ```suggestion
asmctl management config policy-headers list`
```

Reviewer 2: Done! Thanks!
